Misc/reputation-file.json

```diff
@@ -0,0 +1,371 @@
+{
+  "id": "unifiedFileRep",
```

Comment:
if you add it like this then remove https://github.com/demisto/content/blob/indicator-types/Misc/reputation-unifiedFileRep.json